pagopa/pn-frontend
# Disambiguation page - feat/pn 3804

## Short description
Show the disambiguation page when we don't know which user (pf or pg) is logging

## List of changes proposed in this pull request
- Added in Success.tsx an if statement to show the new page or redirect to right url
- Added page test in Success.page.tsx

## How to test
Run login app -> login -> check that the url doesn't have type information and Disambiguation page is shown
